doc/sphinxext/sphinxgallery/__init__.py

```diff
@@ -0,0 +1,8 @@
+"""Sphinx Gallery
+"""
+import os
+__version__ = '0.0.8'
+
+def path_static():
+    """Returns path to packaged static files"""
+    return os.path.abspath(os.path.dirname(__file__))+'/_static'
```

Comment:
PEP8: you should have spaces before and after the "+" operator. 

In addition, it would be better if you used os.path.join, rather than string concatenation (it avoid hard coding the "/", which varies across OS).